### PR TITLE
Update render doc

### PR DESCRIPTION
**Pull request by srschandan**

### CODE REVIEW

File: packages/malloy-render/README.md [hunk 1]
```diff
@@ -21,3 +21,22 @@ Stories are written in the `src/stories` directory. To add more data and Malloy
 ### On Reloading Changes
 
 When running `npm run storybook`, only changes in the malloy-render package will hot reload properly. Changes to dependencies like the core `malloy` package may require a browser reload to work properly.
+
+## Using the Bundled Renderer
+
+Any JS platform with JSON results from a DB query execution and a PreparedResult object from Malloy library (or from Malloy Service) should be able to use the bundled renderer to create an HTML Malloy Result. The renderer bundle is avialable from v0.0.118 onwards. Example usage:
```
Comment (skokenes) [inline packages/malloy-render/README.md:27]:
```suggestion
Any web browser with JSON results from a DB query execution and a PreparedResult object from the Malloy library should be able to use the bundled renderer to create an HTML Malloy Result. The renderer bundle is available from v0.0.118 onwards. Example usage:
```
Small typo, removing reference to internal only stuff...

File: packages/malloy-render/README.md [hunk 2]
```diff
@@ -21,3 +21,22 @@ Stories are written in the `src/stories` directory. To add more data and Malloy
...
+  'https://cdn.jsdelivr.net/npm/@malloydata/render@0.0.118/dist/bundle/bundled_renderer.min.js';
+document.head.appendChild(script);
+var resultElement = document.getElementById('result_div');
+renderMalloyResults(result, total_rows, preparedResult).then(
+  function (malloyResElement) {
+    resultElement.appendChild(malloyResElement);
+  }
+);
+```
+
```
Comment (skokenes) [inline packages/malloy-render/README.md:41]: We might also want to include an example of how to just use the new malloy renderer as a web component? That can come later though

**Review by skokenes — APPROVED** [review]
LGTM

File: packages/malloy-render/README.md (hunk 2 above)
Comment (srschandan) [reply packages/malloy-render/README.md:41]: Ack.

**Pull request merged**